onnxruntime/python/onnxruntime_pybind_mlvalue.cc

```diff
@@ -9,6 +9,7 @@
 #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
 #define PY_ARRAY_UNIQUE_SYMBOL onnxruntime_python_ARRAY_API
 #include <numpy/arrayobject.h>
+#include "numpy_helper.h"
```

Comment:
here and elsewhere, can we use the relative path "python/numpy_helper.h"?

Reply:
Why? "numpy_helper.h" is shorter.

Reply:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes
"All of a project's header files should be listed as descendants of the project's source directory"